src/components/KDropdown/KDropdown.vue

```diff
@@ -29,6 +29,7 @@
             <KButton
               v-if="triggerButtonText"
               :appearance="appearance"
+              :aria-label="isToggled.value ? 'Close dropdown' : 'Open dropdown'"
```

Comment:
I'm not 100% sure this should be hard-coded as the Kongponent has no idea as to the context in which it's being used (e.g. there may be far better text here the app would be able to provide on its own).

e.g. if the dropdown is to "pick a region" and already has a label or other contextual text around the dropdown, adding _more_ labels would actually be harmful to the experience

Reply:
I think you're right here. Decided to go with a different approach